lib/api.js

```diff
@@ -1,8 +1,17 @@
+const _ = require('lodash');
+
 export async function fetchNavigationItemsAPI() {
     const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/navigation-menu-items`)
     const json = await res.json()
     if (json.errors) {
         throw new Error('Failed to fetch API')
     }
-    return json
+
+    const navigationMenuItems = _.sortBy(json, ['sort_order', 'updated_at'])
```

Comment:
This is so that the items appear in the correct order defined by `sort_order` field